Makefile

```diff
@@ -132,6 +139,16 @@ OPENAPI_GEN = $(shell pwd)/bin/openapi-gen
 openapi-gen: ## Download openapi-gen locally if necessary.
 	$(call go-get-tool,$(OPENAPI_GEN),k8s.io/kube-openapi/cmd/openapi-gen@v0.0.0-20211115234752-e816edb12b65)
 
+CLIENT_GEN = $(shell pwd)/bin/client-gen
+LISTER_GEN = $(shell pwd)/bin/lister-gen
+INFORMER_GEN = $(shell pwd)/bin/informer-gen
+.PHONY: client-gen
+client-gen: ## Download client-gen, lister-gen and informer-gen locally if necessary.
```

Comment:
this target name seems a bit misleading; the comment actually reveals its purpose. Can we rename it properly in line with the comment?